<!DOCTYPE html>
<html>
    <head>
    <meta charset="utf-8">
    <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover" name="viewport" />
    <link rel="alternate icon" type="image/png" href="/img/favicon.png">
    <title>Sam Lin | </title>
    
<link rel="stylesheet" href="/css/reset.css">

    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/markdown.css">

    
<link rel="stylesheet" href="/css/fonts.css">

<meta name="generator" content="Hexo 5.4.2"></head>
    <body>
        <div class="paper">
            <div class="paper-main">
                
                    <div class="post-header">
    <a class="logo" href="/">Sam Lin</a>
    <a class="go-home" href="/">
        <svg width="8" height="14" viewBox="0 0 8 14">
            <path d="M7 1L1 7l6 6" stroke="#000" stroke-width="2" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"/>
        </svg>
    </a>
</div>
                
                <div class="post-main">

    
        <div class="post-main-title">
            Sams First Post
        </div>
        <div class="post-meta">
            2022-05-23
        </div>
    

    <div class="post-md">
        <p>&#x3D;&#x3D;Succeessive stages in Gene:&#x3D;&#x3D;</p>
<ul>
<li><p><strong>Genome</strong></p>
</li>
<li><p><strong>Transcriptome</strong></p>
</li>
<li><p><strong>Proteome</strong></p>
</li>
<li><p><strong>Interactome</strong></p>
</li>
<li><p>The maximum number of polypeptide-encoding genes in the genome can be identified directly by characterizing open reading frames (ORFs).</p>
</li>
<li><p>Another approach is to define the number of genes directly in terms of the transcriptome (by directly identifying all the RNAs) or proteome (by directly identifying all the polypeptides).</p>
</li>
</ul>
<p>&#x3D;&#x3D;Genomes Can Be Mapped at Several Levels of Resolution&#x3D;&#x3D;</p>
<ul>
<li>#definition A <strong>linkage map</strong> presents the distance between loci in units based on recombination frequencies.</li>
<li>#definition A <strong>restriction map</strong> is constructed by cutting DNA into fragments with restriction enzymes and measuring the physical distances, in terms of the length of DNA in base pairs (determined by migration on an electrophoretic gel) between the cut sites.</li>
<li>#definition The ultimate genomic map is the <strong>DNA sequence</strong> of the genome.</li>
</ul>
<h1 id="1-Repetitive-and-Non-repetitive-DNA"><a href="#1-Repetitive-and-Non-repetitive-DNA" class="headerlink" title="1 Repetitive and Non-repetitive DNA"></a>1 Repetitive and Non-repetitive DNA</h1><p>&#x3D;&#x3D;The DNA sequences composing a eukaryotic genome can be classified into three groups:&#x3D;&#x3D;</p>
<ol>
<li>Nonrepetitive sequences that are unique</li>
<li>Moderately repetitive sequences that are dispersed and repeated a small number of times, with some copies not being identical</li>
<li>Highly repetitive sequences that are short and usually repeated as tandem arrays</li>
</ol>
<ul>
<li><p>#definition <strong>Nonrepetitive DNA</strong> consists of sequences that are unique: There is only one copy in a haploid genome.</p>
</li>
<li><p>#definition <strong>Repetitive DNA</strong> consists of sequences that are present in more than one copy in each genome.</p>
<ul>
<li><strong>Moderately repetitive DNA</strong> consists of relatively short sequences that are repeated typically 10 to 100x in the genome.</li>
<li><strong>Highly repetitive DNA</strong> consists of very short sequences (typically &lt;100 bp) that are present many thousands of times in the genome, often organized as long regions of tandem repeats</li>
</ul>
</li>
<li><p>#definition <strong>transposons</strong>: a kind of repetitive DNA, short sequences of DNA (up to 5 kb) that have the ability to move to new locations in the genome and&#x2F;or to make additional copies of themselves</p>
</li>
</ul>
<h1 id="2-Identifying-Exons"><a href="#2-Identifying-Exons" class="headerlink" title="2 Identifying Exons"></a>2 Identifying Exons</h1><ul>
<li><p>Sections of DNA that are conserved among species can be identified by performing a zoo blot.</p>
</li>
<li><p>Another technique that allows genomic fragments to be rapidly scanned for the presence of exons is called exon trapping.</p>
</li>
</ul>
<p><img src="/Genes/attachments/Pasted%20image%2020220515105818.png" alt="exon trapping"></p>
<ul>
<li><p>Ultimately, exons can be identified by the large-scale sequencing of cellular mRNAs that is now commonplace</p>
</li>
<li><p>#definition <strong>synteny 共线性</strong>: When pairs of human and mouse homologs are compared, the genes located on either side also tend to be homologs. This relationship is called <strong>synteny</strong>.</p>
</li>
<li><p>#definition <strong>expressed sequence tag (EST)</strong>: A functional gene can be located through the use of an <strong>expressed sequence tag (EST)</strong>, a short portion of a transcribed sequence usually obtained from sequencing one or both ends of a cloned fragment from a cDNA library.</p>
</li>
</ul>
<h1 id="3-The-Conservation-of-Genome-Organization-Helps-to-Identify-Genes"><a href="#3-The-Conservation-of-Genome-Organization-Helps-to-Identify-Genes" class="headerlink" title="3 The Conservation of Genome Organization Helps to Identify Genes"></a>3 The Conservation of Genome Organization Helps to Identify Genes</h1><blockquote>
<p>[!abstract]</p>
<ul>
<li>Methods for identifying functional genes are not perfect and many corrections must be made to preliminary estimates.</li>
<li>Pseudogenes must be distinguished from functional genes.</li>
<li>There are extensive syntenic relationships between the mouse and human genomes, and most functional genes are in a syntenic region.</li>
</ul>
</blockquote>
<p>A functional gene should consist of a series of exons for which:</p>
<ul>
<li>the first exon immediately follows a promoter</li>
<li>the internal exons are flanked by appropriate splicing junctions</li>
<li>the last exon is followed by 3’ processing signals</li>
<li>and a single open reading frame starting with an initiation codon and ending with a termination codon can be deduced by joining the exons together.</li>
</ul>
<p><img src="/Genes/attachments/Pasted%20image%2020220515113155.png"></p>
<p>the mouse has only one functional encoding glyceraldehyde phosphate dehydrogenase gene (GAPDH),   but has ~400 homologous pseudogenes.</p>
<h1 id="4-Some-Organelles-Have-DNA"><a href="#4-Some-Organelles-Have-DNA" class="headerlink" title="4 Some Organelles Have DNA"></a>4 Some Organelles Have DNA</h1><blockquote>
<p>[!Abstract]</p>
<ul>
<li>Mitochondria and chloroplasts have genomes that show non-Mendelian inheritance. Typically they are maternally inherited.</li>
<li>Organelle genomes may undergo somatic segregation in plants.</li>
<li>Comparisons of human mitochondrial DNA suggest that it is descended from a single population that existed ~200,000 years ago in Africa</li>
</ul>
</blockquote>
<h1 id="5-Organelle-Genomes-Are-Circular-DNAs-That-Encode-Organelle-Proteins"><a href="#5-Organelle-Genomes-Are-Circular-DNAs-That-Encode-Organelle-Proteins" class="headerlink" title="5 Organelle Genomes Are Circular DNAs That Encode Organelle Proteins"></a>5 Organelle Genomes Are Circular DNAs That Encode Organelle Proteins</h1><blockquote>
<p>[!Abstract]</p>
<ul>
<li>Organelle genomes are usually (but not always) circular molecules of DNA.</li>
<li>Organelle genomes encode some, but not all, of the proteins used in the organelle.</li>
<li>Animal cell mitochondrial DNA is extremely compact and typically encodes 13 proteins, 2 rRNAs, and 22 tRNAs.</li>
<li>Yeast mitochondrial DNA is 5x longer than animal cell mtDNA because of the presence of long introns.</li>
</ul>
</blockquote>
<p>In mammalian mitochondria, the genome is particularly compact. There are no introns, some genes actually overlap, and almost every base pair can be assigned to a gene. With the exception of the <strong>D-loop</strong>, a region involved with the initiation of DNA replication, no more than 87 of the 16,569 bp of the human mitochondrial genome lie in intergenic regions.</p>
<p><a target="_blank" rel="noopener" href="https://www.39kf.com/cooperate/book/05/molecular-biology/2007-09-25-411479.shtml">D-loop in Mito</a></p>
<h1 id="6-The-Chloroplast-Genome-Encodes-Many-Proteins-and-RNAs"><a href="#6-The-Chloroplast-Genome-Encodes-Many-Proteins-and-RNAs" class="headerlink" title="6 The Chloroplast Genome Encodes Many Proteins and RNAs"></a>6 The Chloroplast Genome Encodes Many Proteins and RNAs</h1><blockquote>
<p>[!Abstract]</p>
<ul>
<li>Chloroplast genomes vary in size, but are large enough to encode 50 to 100 proteins as well as the rRNAs and tRNAs.</li>
</ul>
</blockquote>
<p>The chloroplast genome encodes all the rRNAs and tRNAs needed for protein synthesis in the chloroplast.</p>
<p>Some complexes have have some subunits encoded by the organelle genome and some by the nuclear genome.</p>
<blockquote>
<p>[!example]</p>
<ul>
<li>the gene for the large subunit of ribulose bisphosphate carboxylase (RuBisCO, which catalyzes the carbon fixation reaction of the Calvin cycle), rbcL, is contained in the chloroplast genome; variation in this gene is frequently used as a basis for reconstructing plant phylogenies.</li>
<li>the gene for the small RuBisCO subunit, rbcS, is usually carried in the nuclear genome.</li>
</ul>
</blockquote>
<h1 id="7-Mitochondria-and-Chloroplasts-Evolved-by-Endosymbiosis"><a href="#7-Mitochondria-and-Chloroplasts-Evolved-by-Endosymbiosis" class="headerlink" title="7 Mitochondria and Chloroplasts Evolved by Endosymbiosis"></a>7 Mitochondria and Chloroplasts Evolved by Endosymbiosis</h1><blockquote>
<p>[!Abstract]</p>
<ul>
<li>Both mitochondria and chloroplasts are descended from bacterial ancestors.</li>
<li>Most of the genes of the mitochondrial and chloroplast genomes have been transferred to the nucleus during the organelle’s evolution.</li>
</ul>
</blockquote>
<p>Sequence homologies suggest that mitochondria and chloroplasts evolved separately from lineages that are common with different eubacteria, with mitochondria sharing an origin with <em>a-purple bacteria</em> and chloroplasts sharing an origin with <em>cyanobacteria</em>.</p>
<p>The rate of transfer can be measured directly by introducing a gene that can function only in the nucleus (because it contains a nuclear intron, or because the protein must function in the cytosol) into an organelle.</p>
<h1 id="8-Glossary"><a href="#8-Glossary" class="headerlink" title="8 Glossary"></a>8 Glossary</h1><ul>
<li>#definition <strong>genetic polymorphism</strong>: The coexistence of multiple alleles at a locus in a population is called genetic polymorphism.</li>
<li>#definition <strong>single nucleotide polymorphism (SNP)</strong>: When alleles of the same locus are compared, a difference in a single nucleotide is called a single nucleotide polymorphism (SNP). On average, one SNP occurs for approximately every 1330 bases in the human genome.</li>
<li>#definition <strong>restriction fragment length polymorphism (RFLP)</strong>: A difference in restriction maps between two individuals is called a restriction fragment length polymorphism (RFLP), or “riflip”.</li>
<li>#definition <strong>haplotype 单倍型</strong>: The particular combination of sites found in a specific region is called a haplotype and represents a small portion of the complete genotype.</li>
<li>#definition <strong>DNA fingerprinting</strong>: The use of DNA restriction analysis to identify individuals has been called DNA fingerprinting.</li>
</ul>

    </div>

</div>
                <div class="footer">
    <span>Copyright © 2022 Sam Lin</span>
    <span>Theme Designed By <a target="_blank" href="https://zheli.design/one-paper">這Li</a></span>
</div>


<link rel="stylesheet" href="/css/a11y-dark.min.css">


<script src="/js/highlight.min.js"></script>


<script src="/js/highlightjs-line-numbers.js"></script>


<script>
    hljs.initHighlightingOnLoad();
    hljs.initLineNumbersOnLoad();
</script>

            </div>
        </div>
    </body>
</html>